### PR TITLE
read/write mvp

### PR DESCRIPTION
**Pull request by iberdinsky-skilld**
[Image from Gyazo](https://gyazo.com/0a25d39b6f690fa9575b87f9382833f2)

### CODE REVIEW

File: packages/drupal-yaml/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@skilld/drupal-yaml",
+  "version": "1.0.0",
+  "description": "Drupal YAML reader/writer",
```
Comment (andypost) [inline packages/drupal-yaml/package.json]: If you're using Drupal context, please specify "Drupal theme libraries YAML" (libraries.yml)

File: packages/drupal-yaml/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const yaml = require('js-yaml');
+const fs = require('fs');
+
+const readLibraries = () => {
+  try {
+    const doc = yaml.loadAll(fs.readFileSync('./approach.libraries.yml'));
```
Comment (andypost) [inline packages/drupal-yaml/index.js]: I bet `approach` should be argument

Comment (iberdinsky-skilld) [reply packages/drupal-yaml/index.js]: i will take it from .info file

File: packages/drupal-yaml/package.json (hunk 1 above)
Comment (iberdinsky-skilld) [reply packages/drupal-yaml/package.json]: `+`

**Pull request merged**